cpp/tests/groupby/common/groupby_test_util.hpp

```diff
@@ -32,8 +34,9 @@ inline void test_single_agg(column_view const& keys,
                             column_view const& expect_keys,
                             column_view const& expect_vals,
                             std::unique_ptr<experimental::aggregation>&& agg,
-                            bool ignore_null_keys = true,
-                            bool keys_are_sorted = false,
+                            include_nulls include_null_keys = include_nulls::NO,
+                            bool stable_order = false,
```

Comment:
Any reason will left `stable_order` as a `boolean`?

Reply:
This is the only place it is used and is practically a workaround.